solana/solana-ibc/programs/solana-ibc/src/error.rs

```diff
@@ -59,8 +59,18 @@ pub enum Error {
     /// below required minimum.
     NotEnoughValidators,
 
-    // CPI call from an unidentified program
+    /// CPI call from an unidentified program
     InvalidCPICall,
+
+    /// Unexpected Fee Collector
+    InvalidFeeCollector,
+
+    FeeCollectorChangeProposalNotSet,
+
+    AssetAlreadyExists,
```

Comment:
Could you add docstrings to those as well?

Reply:
sure